cmake/modules/hunter_autotools_project.cmake

```diff
@@ -98,7 +98,7 @@ function(hunter_autotools_project target_name)
 
   if(PARAM_UNPARSED_ARGUMENTS)
     hunter_internal_error(
-        "Invalid arguments passed to hunter_autotools_configure:"
+        "Invalid arguments passed to hunter_autotools_project:"
```

Comment:
We need to retest at least one project that was using this function, like `odb-boost`. In this case INTERNAL_ID of `odb-boost` should be changed.